### PR TITLE
Update RELEASE_PROCESS for tagging

### PR DESCRIPTION
**Pull request by bac**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/69/
Test PASSed.

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -38,10 +44,15 @@ the changes seen while packaging for precise into git.
 Ensure the newly packaged charm deploys and behaves. Ensure you can create a
 new environment.
 
+### Juju 1.25 ###
      export JUJU_DEV_FEATURE_FLAGS=jes
      juju bootstrap
      make deploy
 
+### Juju 2 ###
+     juju bootstrap controller <cloud> --upload-tools
```
Comment (hatched) [inline RELEASE_PROCESS.md]: should controller be `<controller>` as it's user defined as well.

**Comment by hatched** [comment]
:+1: thanks for these updates!

**Comment by jaycee** [comment]
:+1: @bac thanks!

**Comment by bac** [comment]
:shipit:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/70/
Test PASSed.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-charm-merge

**Pull request merged**